### ISSUE TITLE
getCellType returns CELL_TYPE_BLANK for numeric cells

### ISSUE THREAD

**Issue by toreba** [monitorjbl/excel-streaming-reader#23]
In the attached excel file, cell A1 has numeric value 123. The xml contains  <c r="A1"><v>123</v></c>. Since attribute t is missing getCellType returns CELL_TYPE_BLANK. I think CELL_TYPE_NUMERIC should be default when content!=null && type==null in StreamingCell.getCellType

[Simple.xlsx](https://github.com/monitorjbl/excel-streaming-reader/files/55839/Simple.xlsx)

**Comment by monitorjbl** [comment]
Interesting. Let me make a test case for this and get back to you.

**Comment by antone-sb** [comment]
I have this problem too. Seems like attribute "t" is optional, and it can be missing for numeric types. The same goes for StreamingReader.formattedContents: use "n" in switch when getType returns null.

**Comment by frederikb** [comment]
I had a similar issue as @antone-sb with version 0.2.10. I was getting a NPE in ``StreamingReader#unformattedContents`` due to the aforementioned problem. I can confirm that the current snapshot build of 0.2.11 fixes that issue for me.

**Comment by monitorjbl** [comment]
0.2.11 is live in Maven Central, so I'm going to close this issue. Let me know if you still have any issues and I'll re-open it.

**Comment by tjjalava** [comment]
I'm still getting this issue with 0.2.11. Could it be related to the version of Excel? I cloned the repo and ran your tests, and they passed. Then I edited 'data_types.xlsx' and changed the value of the "int"-field, first to string and the back to number -> the tests don't pass anymore (the edited file: [data_types.xlsx](https://github.com/monitorjbl/excel-streaming-reader/files/123246/data_types.xlsx)).

I'm using Excel for Mac 2011, Version 14.3.8.
